endpoints/example.lua

```diff
@@ -7,4 +7,28 @@ return "new-user-id"
 --#TAGS user public
 print("Fetch a given user" .. request.parameters.userId)
 return {id=request.parameters.userId}  -- json by default
+
+--#ENDPOINT GET /api/ratelimit/withauth
+--#TAGS security
+--#SECURITY basic bearer
```

Comment:
```suggestion
--#SECURITY bearer
```
Lets only put one to avoid confusing the reader.